test/marker-index.benchmark.js

```diff
@@ -24,7 +25,13 @@ function runBenchmark () {
     enqueueSplice()
     enqueueDelete()
   }
+
+  for (let i = 0; i < 500; i++) {
+    enqueueRangeQuery()
```

Comment:
I should have benchmarked this from the beginning. Lesson learned. Would be interested in testing different queries even but this is a good start.